images/CSC/temp.md

```diff
@@ -0,0 +1 @@
+asdf
```

Comment:
Do we need this?

Reply:
Deleted. It was used just to create a new directory